src/index.js

```diff
@@ -5,13 +5,22 @@ import { Provider as ReduxProvider } from "react-redux";
 import "./index.css";
 import "./assets/icomoon.css";
 import App from "./App";
+import SandboxApp from './sandbox/SandboxApp';
```

Comment:
double quotes for imports.. you could try `npm run lint:fix' for the common issues.

Reply:
Fixed all lint issues in [commit](https://github.com/singnet/refactored-dapp/pull/110/commits/8ddc786cba95ece44cd6a52e33d7231d5a3244b4)